src/spring/HISTORY.md

```diff
@@ -1,5 +1,10 @@
 Release History
 ===============
+1.7.2
+---
+* Support `--no-wait` in `az spring dev-tool`.
+* Add delete confirmation in `az spring dev-tool` and `az spring application-live-view`.
```

Comment:
Please note that adding confirmation steps will cause breaking change to customers

Reply:
I check the usage in past 30days, there is only internal subscription in our team to use.